test/rules/ordered-imports/grouped-imports/test.ts.lint

```diff
@@ -3,17 +3,17 @@
 import {bar} from '../bar';
 
 import {foo, afoo} from 'foo';
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ [Import sources of different groups must be sorted by: libraries, parent directories, current directory.]
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ [Import sources of different groups must be sorted by: libraries, parent directories, current directory .]
```

Comment:
@adidahiya The reason for this space came up in the first review.  The idea is that if the user configures a group and uses a regex for `match` but doesn't include a `name`, then the name is the regex.  In that case when the order is printed out the user can get confused because the period will show up for the sentence and look like it is part of the regex.  I will see if I can come up with a better error string and automatic name that is more clear.

Reply:
@adidahiya I just pushed up a commit that changes this to remove the space and make the default naming for a group with only a regex be /regex/.

So for example from the tests in `groups-string-list`:

```json
{
    "rules": {
        "ordered-imports": [
            true,
            {
                "import-sources-order": "case-insensitive",
                "named-imports-order": "case-insensitive",
                "grouped-imports": true,
                "groups": ["^app", "^@pkg", "^\\.\\.", "^\\."]
            }
        ]
    }
}
```

you end up with a warning like:

```typescript
import { x } from "@pkg/foo";
import { app_f } from "app/foo";
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ [Imports from this modules are not allowed in this group.  The expected groups (in order) are: /^app/, /^@pkg/, /^\.\./, /^\./.]
```

I think this makes it much more clear that you specified a regex and it is just giving it a default name.  I debated using a different delimiter than `/`, but in the end it seems like this is probably the most familiar to a TS developer as "hey, this is a regex".

Let me know what you think.